src/coreclr/src/nativeaot/System.Private.CoreLib/src/System/Diagnostics/StackFrameExtensions.cs

```diff
@@ -1,61 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Should we keep this, and instead move `StackFrameExtensions.cs` to CoreLib and exclude it for CoreRT?

Reply:
The NativeAOT copy has different implementation of some of the methods, to allow you to get IPs for offline-processing even when the textual symbols are not available. IIRC, this was used by some of crash telemetry libraries for UWP.

These APIs are no-ops in the current shared version.

Reply:
Ah, I see. So we should make S.D.StackTrace typeforward StackFrameExtensions to CoreLib when building with ClrNativeAot = true.

Reply:
I would hope that we can concentrate all these differences in CoreLib, and the rest of the libraries can be identical.

For StackFrameExtensions, it means moving StackFrameExtensions.cs to System.Private.CoreLib for both JIT and NativeAOT.